### DISCUSSION TITLE
Please wait for SENS on loading screen

### DISCUSSION THREAD

**Discussion by jkarolis** [Atlas-OS/Atlas#1231] [Help & Support] [answered]
Hello everyone, after clean reinstall and Atlas deployment on my PC its been giving me "Please wait for SENS" on every bootup (edit: on Windows Loading Screen). Usually (before installing going for Atlas install) it would boot faster than now. I have to wait for like 10-20 seconds before system fully boots to desktop. Do you guys know how should I address this issue ? First time seeing it myself.

**Comment by Firdaus2008** [comment]
i reset my pc after seeing slow bootup like you and its still dosen't solve anything.

**Comment by the-P1neapple** [comment, marked as answer]
@jkarolis The message you are seeing is part of the verbose boot messages that Atlas has applied. This provides more experienced users information on how long it takes to start services such as the 'SENS' service. These messages can be disabled in Atlas Folder > 6. Advanced Configuration > Boot Configuration > Appearance and run Boot Messages.cmd. On the prompt, press 1 to disable boot messages.

From what I am aware, these messages will be disabled by default in the next release. Please note, changing this setting does not let your computer boot faster.

**Reply by Firdaus2008** [reply]
No I still wanted tu see the messages but I want the PC boot faster before this issue🥲

**Reply by jkarolis** [reply]
I've tried this cmd file, but it still shows "Please wait for SENS" just before booting to desktop. Hmm, maybe it will help for someone else
